File: Python/Math/get_biggest_int_in_slice.py
# Get the biggest int in a given slice.
def get_biggest_int_in_slice(given_slice):
    return max(given_slice)


def main():
    # Create a slice of ints from 1 to 10.
    randomSlice = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
    # Get the biggest int in the slice.
    print(get_biggest_int_in_slice(randomSlice))


main()

File: TypeScript/Math/get-biggest-int-in-slice.ts
function main(): void {
    // Create a slice of ints
    let randomValues: number[] = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
    // Get the biggest int in the slice
    console.log(getBiggestIntInSlice(randomValues))
}

main()

// Get the biggest int in a slice of ints
function getBiggestIntInSlice(slice: number[]): number {
    let biggestInt: number = slice[0]
    for (let i: number = 1; i < slice.length; i++) {
        if (slice[i] > biggestInt) {
            biggestInt = slice[i]
        }
    }
    return biggestInt
}